docs/static/releasenotes.asciidoc

```diff
@@ -76,6 +77,67 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
 
+[[logstash-8-17-2]]
+=== Logstash 8.17.2 Release Notes
+
+[[notable-8.17.2]]
+==== Notable issues fixed
+
+* The plugin manager's `update` command now correctly only updates _minor_ versions of plugins by default to avoid breaking changes; if you wish to also include breaking changes, you must now specify `--level=major` https://github.com/elastic/logstash/pull/16974[#16974]
```

Comment:
```suggestion
* The plugin manager's `update` command now correctly updates only _minor_ versions of plugins by default to avoid breaking changes.
If you wish to also include breaking changes, you must specify `--level=major` https://github.com/elastic/logstash/pull/16974[#16974]
```